packages/mdc-typography/_variables.scss

```diff
@@ -25,105 +33,112 @@ $mdc-typography-font-weight-values: (
-    font-weight: map-get($mdc-typography-font-weight-values, light),
-    letter-spacing: -.04em,
-    text-decoration: inherit,
-    text-transform: inherit
-  ),
-  display3: (
-    font-size: 3.5rem, /* 56px */
-    line-height: 3.5rem, /* 56px */
-    font-weight: map-get($mdc-typography-font-weight-values, regular),
-    letter-spacing: -.02em,
-    text-decoration: inherit,
-    text-transform: inherit
-  ),
-  display2: (
-    font-size: 2.8125rem, /* 45px */
-    line-height: 3rem, /* 48px */
-    font-weight: map-get($mdc-typography-font-weight-values, regular),
-    letter-spacing: normal,
-    text-decoration: inherit,
-    text-transform: inherit
-  ),
-  display1: (
-    font-size: 2.125rem, /* 34sp */
-    line-height: 2.5rem, /* 40sp */
-    font-weight: map-get($mdc-typography-font-weight-values, regular),
-    letter-spacing: normal,
-    text-decoration: inherit,
-    text-transform: inherit
-  ),
-  headline: (
-    font-size: 1.5rem, /* 24sp */
-    line-height: 2rem, /* 32sp */
-    font-weight: map-get($mdc-typography-font-weight-values, regular),
-    letter-spacing: normal,
-    text-decoration: inherit,
-    text-transform: inherit
-  ),
-  title: (
-    font-size: 1.25rem, /* 20sp */
-    line-height: 2rem, /* 32sp */
-    font-weight: map-get($mdc-typography-font-weight-values, medium),
-    letter-spacing: .02em,
-    text-decoration: inherit,
-    text-transform: inherit
-  ),
-  subheading2: (
-    font-size: 1rem, /* 16sp */
-    line-height: 1.75rem, /* 28sp */
-    font-weight: map-get($mdc-typography-font-weight-values, regular),
-    letter-spacing: .04em,
-    text-decoration: inherit,
-    text-transform: inherit
-  ),
-  subheading1: (
-    font-size: .938rem, /* 15sp */
-    line-height: 1.5rem, /* 24sp */
-    font-weight: map-get($mdc-typography-font-weight-values, regular),
-    letter-spacing: .04em,
-    text-decoration: inherit,
-    text-transform: inherit
-  ),
-  body2: (
-    font-size: .875rem, /* 14sp */
-    line-height: 1.5rem, /* 24sp */
-    font-weight: map-get($mdc-typography-font-weight-values, medium),
-    letter-spacing: .04em,
-    text-decoration: inherit,
-    text-transform: inherit
-  ),
-  body1: (
-    font-size: .875rem, /* 14sp */
-    line-height: 1.25rem, /* 20sp */
-    font-weight: map-get($mdc-typography-font-weight-values, regular),
-    letter-spacing: .04em,
-    text-decoration: inherit,
-    text-transform: inherit
-  ),
-  caption: (
-    font-size: .75rem, /* 12sp */
-    line-height: 1.25rem, /* 20sp */
-    font-weight: map-get($mdc-typography-font-weight-values, regular),
-    letter-spacing: .08em,
-    text-decoration: inherit,
-    text-transform: inherit
-  ),
-  button: (
-    font-size: .875rem, /* 14sp */
-    line-height: 2.25rem, /* 36sp */
-    font-weight: map-get($mdc-typography-font-weight-values, medium),
-    letter-spacing: .04em,
-    text-decoration: none,
-    text-transform: uppercase
+$mdc-typography-styles: mdc-set-typography-styles_(
```

Comment:
Remove the `!default` at the end of this map; we're exposing alternate means of changing it now

Reply:
I'm under the impression that the user should also be able to override the entire map. They could override each individual style, but it seems easier to just leave this default.

Reply:
I guess that's fair; I'm just a bit concerned about effectively having 2 ways to do the same thing now. It's good that we're recommending the global variables to patch as little as necessary though.